android/app/build.gradle

```diff
@@ -13,62 +16,62 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
-    defaultConfig {
-        applicationId "org.jetbrains.kotlinconf"
-        minSdkVersion 18
-        targetSdkVersion 26
-        versionCode 7
-        versionName "1.0.6"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
+  compileSdkVersion 28
+  defaultConfig {
+    applicationId "org.jetbrains.kotlinconf"
+    minSdkVersion 18
+    targetSdkVersion 28
+    versionCode 7
+    versionName "1.0.6"
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+  }
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
+  }
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:26.1.0'
-    compile 'com.android.support:recyclerview-v7:26.1.0'
-    compile 'com.android.support.constraint:constraint-layout:1.0.2'
-    testCompile 'junit:junit:4.12'
-    androidTestCompile('com.android.support.test.espresso:espresso-core:3.0.1', {
-        exclude group: 'com.android.support', module: 'support-annotations'
-    })
+  implementation fileTree(dir: 'libs', include: ['*.jar'])
+  implementation 'com.android.support:appcompat-v7:28.0.0-beta01'
+  implementation "com.android.support:support-v4:28.0.0-beta01"
+  implementation 'com.android.support:recyclerview-v7:28.0.0-beta01'
+  implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+  testImplementation 'junit:junit:4.12'
```

Comment:
For the libraries we're using across the modules, should we extract libraries versions to an external variable to keep consistency across modules and have one place where we change it (like with `kotlin_version`)?

Reply:
Yeap